### ISSUE TITLE
Don't use batch_normalization

### ISSUE THREAD

**Issue by gauss-clb** [hwalsuklee/tensorflow-generative-model-collections#20]
See [here](https://github.com/hwalsuklee/tensorflow-generative-model-collections/blob/master/ops.py#L22), `updates_collections=None` which means `moving_mean` and `moving_var` won't be put into `tf.GraphKeys.UPDATE_OPS`. so [tf.get_collection(tf.GraphKeys.UPDATE_OPS)](https://github.com/hwalsuklee/tensorflow-generative-model-collections/blob/master/GAN.py#L117) is `[]`,  I think it's a bug.

But if you just delete `updates_collections=None`, it can't work.  Because there are two graphs for discriminators and there are different `moving_mean` and `moving_var` for different graph.  So I think you should use different `tf.control_dependencies` for discriminator and generator.

**Comment by Netzeband** [comment]
In the paper about WGAN-GP the authors stated to not use Batch-Normalization at all in the Discriminator, since it will violate some assumptions which were made to archive Lipschitz-1-Criteria by using gradient penalty. 

Since the authors of this paper here do not use Wasserstein loss at all, Lipschitz-1-Criteria should not be an issue anymore. However maybe the authors can give some practical advices about using BN in discriminator of DRAGAN.
